lambdas/answer/src/test/java/uk/gov/di/ipv/cri/kbv/api/handler/QuestionAnswerHandlerTest.java

```diff
@@ -205,6 +211,7 @@ void shouldReturn200WhenNextSetOfQuestionsAreReceivedFromExperian() throws IOExc
         when(mockObjectMapper.writeValueAsString(questionStateMock)).thenReturn("question-state");
         when(questionStateMock.hasAtLeastOneUnAnswered()).thenReturn(false);
         when(mockKBVGateway.submitAnswers(any())).thenReturn(questionsResponseMock);
+        doNothing().when(mockKBVAnswerStorageService).save(questionsResponseMock);
```

Comment:
If this line is removed does the test still pass ?
If so, this line can be removed